jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClientManager.java

```diff
@@ -314,6 +314,14 @@ public boolean removeMucClient(String id)
        return true;
     }
 
+    /**
+     * Stops and removed all MucClients.
+     */
+    public void stop()
```

Comment:
Is this totally new?  I don't see this logic relocated from anywhere here or in the JVB pr

Reply:
Yes, it's new. I discovered the bridge never stops the clients, though there's [code that intends to do that](https://github.com/jitsi/jitsi-videobridge/blob/master/src/main/java/org/jitsi/videobridge/xmpp/ClientConnectionImpl.java#L115)